src/main/java/org/opentripplanner/updater/trip/TimetableSnapshotSource.java

```diff
@@ -347,6 +370,26 @@ public UpdateResult applyTripUpdates(
     return updateResult;
   }
 
+  /**
+   * This shouldn't be used outside of this class for other purposes than testing where the forced
+   * snapshot commit can guarantee consistent behaviour.
+   */
```

Comment:
Is it correct that the only change to this method was adding this Javadoc and removing the private access modifier? So this is the same method body that used to be lower in the same file, but it has been moved with only comments added and "private" removed?

Note that when a method is both moved and changed, it can be difficult for a reviewer to tell what was changed. The reader basically has to look through line by line and character by character.

Reply:
Yes this is correct. It's an unfortunately side-effect of our code conventions that changing the access modifier can lead to the method being moved which makes the review more difficult.